apps/drupal/templates/field/image.html.twig

```diff
@@ -0,0 +1,25 @@
+{#
+/**
+ * @file
+ * Theme override of an image.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the img tag.
+ * - style_name: (optional) The name of the image style applied.
+ *   We're using as a class below. Set you style in Drupal to a
+ *   Bootstrap style for automatic styling.
+ *   See https://getbootstrap.com/docs/4.0/content/images/
+ *
+ * @see template_preprocess_image()
+ */
+#}
+
+{% set image_classes = [
+  'm-3',
```

Comment:
Some of these look like very specific classes add to all images (m-3, float-right).  This file name should probably be updated to the field you are trying to affect here.